system/codesigner/gpgcodesigner.cpp

```diff
@@ -15,9 +15,12 @@
     limitations under the License.
 ############################################################################## */
 
+#include <sys/stat.h>
```

Comment:
Is this still needed?  I'm slightly surprised it didn't cause problems building on windows.